react/content-list.jsx

```diff
@@ -45,14 +58,30 @@ let ContentList = React.createClass({
     return this.setStateFromDattCore()
   },
 
+  componentWillUnmount: function () {
+    this.unmonitorDattCore()
+  },
+
   componentWillReceiveProps: function () {
     return this.setStateFromDattCore()
   },
 
   monitorDattCore: function () {
     let dattcore = this.props.dattcore
+    let uiEvents = this.props.uiEvents
+
     dattcore.on('peers-content-auth', this.handlePeersContentAuth)
     dattcore.on('content-content-auth', this.handleContentContentAuth)
+    uiEvents.on('refresh-content', this.handleRefreshContent)
```

Comment:
Since making this comment, I've looked into it a bit, and redux seems to be a good tool to use for this instead of flux. Redux is also what's used by "MERN", a tool I would like to use to replace our landing page and blog. So using Redux in datt itself would also be good for consistency.